.changeset/brave-jeans-rule.md

```
@@ -0,0 +1,5 @@
+---
+"@siemens/ix": patch
+---
+
+Color **ix-blind** borders accorindg to variant.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)